listenbrainz/db/feedback.py

```diff
@@ -0,0 +1,152 @@
+import sqlalchemy
+
+from listenbrainz import db
+
+
+def insert(user_id, recording_msid, score):
+    """ Inserts a feedback record for a user's loved/hated recording into the database.
+        If the record is already present for the user, the score is updated to the new
+        value passed.
+        Args:
+            user_id (int): the row id of the user
+            recording_msid (string): the MessyBrainz ID of the recording
+            score (int): The score associated with the recording (+1/-1 for love/hate respectively)
```

Comment:
where will validation on this value happen? do other numbers have meaning? -10? 0? 20? Do we want to limit the ranges here at all?

Reply:
I was thinking to validate the values at the API endpoint function to prevent unnecessary db calls if the value is not in [-1,01].

Reply:
Added a validator check in the class.